app/view_models/base_view_model.rb

```diff
@@ -36,10 +36,11 @@ def build
     private
 
     def params(attributes)
-      claim.attributes
-           .merge(claim.data)
-           .merge(attributes, 'claim' => claim)
-           .slice(*klass.attribute_names)
+      key = crime_application.is_a?(Claim) ? 'claim' : 'application'
```

Comment:
I feel like this would get confusing and would prefer just a generic key like `submitted` or `model`

Reply:
Good thinking. Done.